### ISSUE TITLE
Move att.translatable to from tei to tagdocs module 

### ISSUE THREAD

**Issue by ebeshero** [TEIC/TEI#2097]
In response to https://github.com/TEIC/TEI/issues/2036, VF2F agrees with @peterstadler  that we should move `att.translatable` to the tagdocs module, to help effectively guide people to better understanding of how to use `@versionDate` appropriately. We're opening this ticket b/c this requires some testing.

**Comment by sydb** [comment]
Commit 9d8122417 moves att.translatable from ST to TD (directly on dev branch). If Mr. Jenkins approves, this ticket can be closed.

**Comment by sydb** [comment]
Problems are legion. The first is that lots of files in Test/ actually have a `@versionDate` on an element that is not a descendant of a *Spec element:

- Test/testplace-kml.xml: The corresponding ODD does **not** include tagdocs, so I removed the `@versionDate`
- Test/testall.xml: The corresponding ODD does include tagdocs, so this is an appropriate test of using `@versionDate` in a silly place
- Test/testplace-nogml.xml: This file does not have a corresponding ODD and is not mentioned in the Makefile; thus I do not think it is actually part of our testing procedure and left it alone
- Test/testplace.xml: This file _does_ have a corresponding ODD, but is not mentioned in the Makefile; there are 18 `@versionDate`s, so just in case it ever gets used I added tagdocs to the ODD file.
-  Test/testspoken.xml: The corresponding ODD does not have tagdocs, so I removed the `@versionDate`s (and `@xml:id`s)

Also, not surprisingly, in several places the att.translatable.attributes pattern was being referenced before it had been defined. Thus I added predeclare="true".

It _looks_ like it is building OK again. Let’s see what Mr. Jenkins says this time …
